WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -62,6 +60,12 @@ import WordPressFlux
         return ReaderPostService(managedObjectContext: syncContext)
     }()
 
+    /// An alias for the apps's main context – temporarily replaces  `newMainContextChildContext` until we have `NSPersistentContainer` support
+    ///
+    private var viewContext: NSManagedObjectContext {
```

Comment:
This is a shortcut to ensure that it breaks properly once we migrate this class to using a DI-ed Core Data Stack – if we referenced the `sharedInstance` directly it'd be easy to miss an instance.